### PR TITLE
feat: palm support

### PR DESCRIPTION
**Pull request by Henrrypg**
# Description

This PR does some changes to dockerimage in order to support mong4.4 for palm tutor version. it also adds minio init job to create backups bucket

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]
LGTM

**Review by MoisesGSalas — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ classifiers = [
 requires-python = ">=3.8"
 
 dependencies = [
-    "tutor>=18.0.0,<19.0.0"
+    "tutor>=16.0.0,<17.0.0"
```
Comment (jfavellar90) [inline pyproject.toml:30]: Should we change this [version](https://github.com/eduNEXT/drydock-backups/blob/hpg/palm-support/drydock_backups/__about__.py) value to start from 16? or is it automatic?

Comment (Henrrypg) [reply pyproject.toml:30]: It is automatic once we run the release

**Review by jfavellar90 — APPROVED** [review]

File: drydock_backups/plugin.py [hunk 2]
```diff
@@ -23,7 +23,9 @@
         "AZURE_ACCOUNT_NAME": "",
         "CUSTOM_STORAGE_ENDPOINT": None,
         "K8S_USE_EPHEMERAL_VOLUMES": False,
+        "K8S_CUSTOM_STORAGE_CLASS": None,
```
Comment (jfavellar90) [inline drydock_backups/plugin.py]: I would name this variable K8S_EPHEMERAL_VOLUME_STORAGE_CLASS

File: drydock_backups/plugin.py [hunk 3]
```diff
@@ -23,7 +23,9 @@
...
         "K8S_EPHEMERAL_VOLUME_SIZE": "8Gi",
+        "MINIO_EXPIRATION_DAYS": 0,
```
Comment (jfavellar90) [inline drydock_backups/plugin.py:28]: Could you please add to the readme the description of the variables you added?

**Review by jfavellar90 — APPROVED** [review]

**Pull request merged**